astropy/coordinates/builtin_frames/lsr.py

```diff
@@ -114,26 +109,21 @@ def lsr_to_icrs(lsr_coord, icrs_frame):
 
 @format_doc(base_doc, components=doc_components_gal, footer=doc_footer_lsr)
 class GalacticLSR(BaseCoordinateFrame):
-    r"""A coordinate or frame in the Local Standard of Rest (LSR), axis-aligned
-    to the Galactic frame.
-
-    This coordinate frame is axis-aligned and co-spatial with
-    `~astropy.coordinates.ICRS`, but has a velocity offset relative to the
-    solar system barycenter to remove the peculiar motion of the sun relative
-    to the LSR. Roughly, the LSR is the mean velocity of the stars in the solar
-    neighborhood, but the precise definition of which depends on the study. As
```

Comment:
Same here, but I'd actually add explanation about which axes are aligned. It says below "This frame is axis-aligned and co-spatial with the `~astropy.coordinates.Galactic` frame."
I think here concision < explanation.